spec/indexers/dataset_indexer_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe DatasetIndexer do
+  subject(:solr_document) { service.generate_solr_document }
+  let(:service) { described_class.new(work) }
+  let(:work) { create(:dataset) }
```

Comment:
I wonder if this would work with 'new' instead of 'create'.  If so, that might make it a bit more efficient since it won't have to persist the object.  But I won't hold it up for that.

Reply:
@coblej Good idea, thanks. I just tried that but got some errors that I can't quite figure out how to get around:
```
Failure/Error: let(:work) { new(:dataset) }
       `new` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
```

Reply:
@seanaery Now that I think about it, I realize I'm not sure what `create` is actually being called on, so my suggestion may not make any sense.  Since what you've got seems to work, I'd probably just leave it as you have it.